### ISSUE TITLE
[Bug]: Generating exception `Rejecting promise with error: Null check operator used on a null value` through `flutter build web`

### ISSUE THREAD

**Issue by yangfanyu** [Baseflow/flutter-permission-handler#1237]
### Please check the following before submitting a new issue.

- [X] I have searched the [existing issues](https://github.com/baseflow/flutter-permission-handler/issues).
- [X] I have carefully [read the documentation](https://github.com/Baseflow/flutter-permission-handler/blob/main/permission_handler/README.md) and verified I have added the required platform specific configuration.

### Please select affected platform(s)

- [X] Android
- [X] iOS
- [X] Windows

### Steps to reproduce

if i use `permission_handler: ^10.0.0`, then run `flutter build web` ,  every thing is ok:

![1700817040668](https://github.com/Baseflow/flutter-permission-handler/assets/17940584/aa98c380-1e33-490d-ae07-c276360e91af)

if i use `permission_handler: ^11.0.0`, then run `flutter build web` ,  error:

![1700817100888](https://github.com/Baseflow/flutter-permission-handler/assets/17940584/bbae58e6-6a69-4eb0-b0e1-9c702e4610ab)

then i check the source code in `permission_handler_html.dart` :
```dart
  
class WebPermissionHandler extends PermissionHandlerPlatform {
  /// Maybe the following line of code causes the error
  /// I try modify it to "static final html.MediaDevices? _devices = html.window.navigator.mediaDevices;" 
 /// then every thing is ok
  static final html.MediaDevices _devices = html.window.navigator.mediaDevices!;
...
}
```

### Expected results

...

### Actual results

...

### Code sample

...

### Version

11.0.0+

### Flutter Doctor output

...

**Comment by rooney0502** [comment]
Same problem, i use flutter 3.16 and permission_handler: ^11.1.0. It seems that only the web has the problem

**Comment by TimHoogstrate** [comment]
Dear @rooney0502 or @yangfanyu,

Does it also happen in the supplied example app? Or did you only add the permission handler and then just run `flutter build web` without really using it in the code?

Kind regards,

**Comment by yangfanyu** [comment]
thanks, it fix on `permission_handler_html 0.1.0+2` :
from 
```dart
....
    static final html.MediaDevices _devices = html.window.navigator.mediaDevices!;
...
``` 
to 

```dart
....
    static final html.MediaDevices? _devices = html.window.navigator.mediaDevices!;
...
``` 
it works ok.

Kind regards. @TimHoogstrate
